package.json

```diff
@@ -14,6 +14,8 @@
   "scripts": {
     "start": "cd infrastructure/develop && ./compose-build.sh && ./run-servers.sh dev-build",
     "prod-start": "cd infrastructure/develop && ./compose-build.sh && ./run-servers.sh prod-build",
+    "no-docker-start:frontend": "./no_docker_frontend_start.sh",
+    "no-docker-start:backend": "./no_docker_backend_start.sh",
```

Comment:
I just got the new macbook pro with the M1 chip so it doesn't support docker. So we can use this for people who can't/don't want to use docker and just use rails with postgres natively.